### ISSUE TITLE
factory 4.1.1 slows down factoring immensely

### ISSUE THREAD

**Issue by DanGrayson** [Macaulay2/M2#783]
Reported by Doug Leonard:
```
P=QQ[x,y,z,MonomialOrder=>Lex];
d=z^4+z^2*x*y^9+z*x^9*y+x^5*y^5;
phi=map(P,P,matrix{{x^13*y^4,x^3*y,x^20*y^6*z}});
time f=factor(phi(d));
```

The fix seems to be to go back to using factory 4.1.0 instead of 4.1.1.

**Comment by DanGrayson** [comment]
Resolved by 531b695004f0f3c674d7606eca3cb52eb676bd54

**Comment by DanGrayson** [comment]
reopening -- we should figure this out

**Comment by DanGrayson** [comment]
Resolved finally by commit 50a8bb75cdf5a51e0d54539570d78efa7fc8aae2
